tools/mkldscript.c

```diff
@@ -249,8 +250,16 @@ static void write_ld_script(FILE *fout)
         // DWARF 3
           "    .debug_pubtypes 0 : { *(.debug_pubtypes) }"                                  "\n"
           "    .debug_ranges   0 : { *(.debug_ranges) }"                                    "\n"
-        // DWARF Extension
+        // DWARF 5
```

Comment:
The comments mention DWARF 3 and 5. DWARF 4 did not add any other section?
Also, DWARF 6 is in development, maaaayybe we want to keep an eye on that?

Reply:
As far as I've been able to find, DWARF 4 doesn't seem to add any new ELF sections.

We can worry about DWARF 6 once compilers start emitting it and stuff breaks again 😢